api/v1beta2/bgppeer_types.go

```diff
@@ -17,10 +17,14 @@ limitations under the License.
 package v1beta2
 
 import (
+	"time"
+
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const DefaultConnectTime = 120 * time.Second
```

Comment:
this constant isn't used here but is used in other places for example in the conversion function to the v1beta1 bgpeer version, the reason i chose to put it in this file because this value already appears in this file as part of the kubebuilder comments and logically its not an feature in the v1beta1 version so i thought it be better to put in here even if its not used in this file

Reply:
Not relevant anymore due to making connect time pointer field